dolfyn/tools/misc.py

```diff
@@ -336,14 +293,13 @@ def interpgaps(a, t, maxgap=np.inf, dim=0, extrapFlg=False):
                      a[gd[inds[i2]]]).astype(a.dtype)
 
 
-def medfiltnan(a, kernel, thresh=0):
```

Comment:
This one is also useful to expose to the API.

Reply:
Done